### ISSUE TITLE
M1 (Apple Silicon) Install Fails with go-serial

### ISSUE THREAD

**Issue by rymcol** [la5nta/pat#272]
On `go get github.com/la5nta/pat` you get:

```# github.com/albenik/go-serial/v2
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:20:23: undefined: tcCMSPAR
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:25:23: undefined: tcCMSPAR
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:30:23: undefined: tcCMSPAR
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:33:6: undefined: tcCMSPAR
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:38:22: undefined: tcCMSPAR
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:41:6: undefined: tcCMSPAR
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:46:22: undefined: tcCMSPAR
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:55:18: undefined: databitsMap
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:82:22: undefined: tcCRTSCTS
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:84:23: undefined: tcCRTSCTS
../../../../pkg/mod/github.com/albenik/go-serial/v2@v2.3.0/termios_unix.go:84:23: too many errors```

I'm assuming something is missing for arm somewhere, but I could not find any other tickets here or there

**Comment by rymcol** [comment]
I am VA3RYC, and I am a professional go dev if you want more than that feel free to poke me

**Comment by martinhpedersen** [comment]
Thanks for reporting this 🙂 

It looks like github.com/albenik/go-serial does not compile on arm64 then. I will file a bug report 👍

**Comment by martinhpedersen** [comment]
I've submitted PR https://github.com/albenik/go-serial/pull/25 to resolve this upstream.

**Comment by martinhpedersen** [comment]
Resolved in v0.12.1.
